common/coconut-interface/src/lib.rs

```diff
@@ -0,0 +1,170 @@
+pub use coconut_rs::*;
+use coconut_rs::{Attribute, Base58, BlindSignRequest, BlindedSignature, PublicKey};
+use getset::{CopyGetters, Getters};
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Getters, CopyGetters, Clone)]
+pub struct Credential {
+    #[getset(get = "pub")]
+    n_params: u32,
+    #[getset(get = "pub")]
+    theta: Theta,
+    public_attributes: Vec<String>,
```

Comment:
Why are the attributes Strings?

Reply:
We can change that in the future, once we know more of how an attribute should look like.

Reply:
Well, an attribute is a Scalar : ) There's nothing more to it really

Reply:
Right. The reason Drazen went with String is probably because it's an easier representation, using base 58, in the context of (de)serializing it for the `InitMessage`. Going deeper into `coconut-rs` and implementing (de)serializing for `Attributes` is I guess another possibility.